frontend/packages/dev-console/integration-tests/support/commands/elementInteractions.ts

```diff
@@ -0,0 +1,51 @@
+export {}; // needed in files which don't have an import to trigger ES6 module usage
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      clickNavLink(path: [string, string]): Chainable<Element>;
+      selectByDropDownText(selector: string, dropdownText: string): Chainable<Element>;
+      mouseHover(selector: string): Chainable<Element>;
+      selectValueFromAutoCompleteDropDown(
+        selector: string,
+        dropdownText: string,
+      ): Chainable<Element>;
+    }
+  }
+}
+
+Cypress.Commands.add('clickNavLink', (path: [string, string]) => {
```

Comment:
Hi @makambalaji, Another reason to use common/shared code for DRY best practices.  This cmd is part of a larger, more defined [nav](https://github.com/openshift/console/blob/52986b6fca9befc99af0581339ec004b201db13c/frontend/packages/integration-tests-cypress/views/nav.ts#L26) helper object, which can handle a single [nav](https://github.com/openshift/console/blob/8e3640ec2912ca88fb6ce7c1ddeae3ea03a5515e/frontend/packages/integration-tests-cypress/support/nav.ts#L12) section.  In other words, you are redoing what has already been implemented.